tests/graphql/test_graphql_client.py

```diff
@@ -0,0 +1,57 @@
+import mock
+
+import kili
+from kili.graphql.graphql_client import GraphQLClient, GraphQLClientName
+
+
+def test_graphql_client(mocker):
+    response = mock.MagicMock()
+    response.json = lambda: {"version": "1.0.0"}
+
+    mocker.patch("kili.graphql.graphql_client.RequestsHTTPTransport")
+    mocker.patch("kili.graphql.graphql_client.requests.get", return_value=response)
+    mocker.patch("kili.graphql.graphql_client.print_schema", return_value="")
+    mocker.patch("kili.graphql.graphql_client.Client")
+    mocker.patch("kili.graphql.graphql_client.gql")
+    mocker.patch("kili.graphql.graphql_client.__version__", "1.0.0")
+    mocker.patch("kili.graphql.graphql_client.python_version", return_value="4.0.0")
+
+    graphql_client = GraphQLClient("http://dummy-endpoint/", "dummy_api_key", GraphQLClientName.SDK)
+
+    kili.graphql.graphql_client.RequestsHTTPTransport.assert_called_once_with(
+        url="http://dummy-endpoint/",
+        headers={
+            "Authorization": f"X-API-Key: dummy_api_key",
+            "Accept": "application/json",
+            "Content-Type": "application/json",
+            "apollographql-client-name": "python-sdk",
+            "apollographql-client-version": "1.0.0",
+            "kili-client-name": "python-sdk",
+            "kili-client-version": "1.0.0",
+            "kili-client-language-name": "Python",
+            "kili-client-language-version": "4.0.0",
+        },
+        cookies=None,
+        auth=None,
+        use_json=True,
+        timeout=30,
+        verify=True,
+        retries=20,
+        method="POST",
+    )
+    graphql_client.included_logged_method_locations = ["test_graphql_client"]
+    graphql_client.execute(query="dummy", variables={})
+
+    assert graphql_client._gql_client.execute.call_args[1]["extra_args"] == {
+        "headers": {"kili-client-method-name": None}
+    }
```

Comment:
don't we add a test to test that we have the right kili-client-method-name called ?